dxm/src/main/java/io/pcp/parfait/dxm/MmvWritable.java

```diff
@@ -2,8 +2,9 @@
 
 import java.nio.ByteBuffer;
 
-public interface MmvWritable {
+interface MmvWritable {
 
+    int getOffset();
```

Comment:
not sure why this one had to pop up here, I really liked the previous commits making this an internal representation that the visitor pattern of the `writeToMmv` just uses.

Hard to see from this commit only why it's here